extensions/websockets-next/deployment/src/main/java/io/quarkus/websockets/next/deployment/WebSocketProcessor.java

```diff
@@ -946,6 +992,30 @@ private static void generateOnMessage(ClassCreator endpointCreator, WebSocketEnd
         }
     }
 
+    private static ResultHandle callBusinessMethod(ClassCreator clazz, MethodCreator constructor, Callback callback,
+            String messageType, BytecodeCreator bytecode, ResultHandle beanInstance, ResultHandle[] args,
+            InvokerFactoryBuildItem invokerFactory) {
+        if (KotlinUtils.isKotlinSuspendMethod(callback.method)) {
```

Comment:
It would not hurt to add a short description what exactly is happening here. I.e. a field is created, invoker is stored, etc.

Reply:
Good point.

Reply:
Done.